string_test.go

```diff
@@ -213,341 +213,611 @@ func TestString_IsEmpty(t *testing.T) {
 }
 
 func TestString_IsEqual(t *testing.T) {
-	reporter := newMockReporter(t)
-
-	value := NewString(reporter, "foo")
-
-	assert.Equal(t, "foo", value.Raw())
-
-	value.IsEqual("foo")
-	value.chain.assertNotFailed(t)
-	value.chain.clearFailed()
-
-	value.IsEqual("FOO")
-	value.chain.assertFailed(t)
-	value.chain.clearFailed()
+	t.Run("basic", func(t *testing.T) {
```

Comment:
Could you please remove subtests from tests that have only single subtest?

Here and in other tests.

Reply:
I see, sry I forgot to remove that one.

Will update.

Reply:
Updated https://github.com/gavv/httpexpect/pull/331/commits/03c3402010022b6205d4e9e5b4e7db78b6896595